examples/gallery/plot/logo.py

```diff
@@ -0,0 +1,19 @@
+"""
+Logo
+----
+
+The :meth:`pygmt.Figure.logo` method allows to place the GMT logo on a map.  
+
+For more advanced options, see the full option list at :gmt-docs:`logo.html`.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 10, 0, 2], projection="X6c", frame=True)
+
+# add the GMT logo in the upper right corner of the current map,
+# scaled up to be 3 cm wide and offset by 0.3 cm from the border
+fig.logo(position="jTR+o0.3c/0.3c+w3c")
```

Comment:
If only one value is provided for the offset it is applied in both the x and y directions. I would recommend either having different values for the two axes to illustrate how the offset can be set for multiple axes. Alternatively, use `position="jTR+o0.3c+w3c"` since it has the same effect.